code/script.js

```diff
@@ -1,12 +1,843 @@
+        chat.innerHTML += `
+          <div class="result-container">
+            <div class="result-list">
+              <h3>Scarecrow</h3>
+              <ul>
+                What you need:
+                <li>Blue jeans or overalls</li>
+                <li>Plaid flannel shirt</li>
+                <li>Face paint</li>
+                <li>Straw hat</li>
+              </ul>
+              <img class="img" src="https://i.ytimg.com/vi/EZk1u_Yiw3o/maxresdefault.jpg"/>
+            </div>
+          </div>
+          <button class="try-again" onClick="window.location.reload();">Try again</button>
+        `
+      });
+      document.getElementById('store-bought').addEventListener("click", () => {
+        showMessage("Store-bought", 'user');
+        showMessage(`Ladybug or Monkey`, 'bot');
+        chat.innerHTML += `
+          <div class="result-container">
+            <div class="result-list">
+              <h3>Ladybug</h3>
+              <ul>
+                What you need:
+                <li>
+                  <a target="_blank" href="https://www.amazon.co.uk/Amscan-International-Darling-Ladybug-Costume/dp/B00TIVPXDQ">Buy here</a>
+                </li>
+              </ul>
+            </div>  
+            <div class="result-list">
+              <h3>Monkey</h3>
+              <ul>
+                What you need:
+                <li>
+                  <a target="_blank" href="https://www.amazon.com/Sqlszt-Animal-Cosplay-Pajamas-Costume/dp/B00ZI5ILY4/ref=sr_1_1_sspa?dchild=1&keywords=adult+monkey+costume&qid=1612440212&sr=8-1-spons&psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUExT0JHVUs0SVo5QUZMJmVuY3J5cHRlZElkPUEwOTAwMTYxM0swU0U1RjU3SjdVRiZlbmNyeXB0ZWRBZElkPUEwNzcxMjU1NlFLUkFRNUlCSDVEJndpZGdldE5hbWU9c3BfYXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ==">Buy here</a>
+                </li>
+              </ul>
+            </div>  
+          </div>
+          <button class="try-again" onClick="window.location.reload();">Try again</button>
+        `
+      });
+      document.getElementById('a-combo').addEventListener("click", () => {
+        showMessage("A combo", 'user');
+        showMessage(`Beanie baby`, 'bot');
+        chat.innerHTML += `
+          <div class="result-container">
+            <div class="result-list">
+              <h3>Beanie baby</h3>
+              <ul>
+                What you need:
+                <li>Animal ears</li>
+                <li>Matching colorful clothes or onepiece</li>
+                <li>Hommade "ty" heart tag</li>
+              </ul>
+              <img class="img" src="https://media.glassdoor.com/l/81/b9/03/0f/the-beanie-babies-of-nurses-prn.jpg"/>
+            </div>
+          </div>
+          <button class="try-again" onClick="window.location.reload();">Try again</button>
+        `
+      });      
+
+
+    });
+
+  } else {
+    showMessage(`Please answer the question, what style are you looking for? Funny, scary or cute?`, 'bot');
+    step -= 1;
+  }
+}
```

Comment:
One of the main purposes of the function is to store blocks of code that you can reuse.  
This function is 816 lines long so it for sure can be cut to few smaller functions. 

Please try to rewrite it and if you meet any problems you can't solve by yourself you can always ask me on Slack.